gen.py

```diff
@@ -36,6 +36,7 @@
     'commons',
     'couchdb',
     'lucene',
+    'solr'
```

Comment:
Is there a lacking `,` at the end here, and may that possibly be why Justin still cannot commit?

Reply:
I tested this code in local python and the resulting dictionary is `{'zookeeper', 'commons', 'lucene', 'abdera', 'couchdb', 'calcite', 'solrtrafficcontrol', 'camel', 'bookkeeper'}`  ...